microceph/cmd/microceph/init.go

```diff
@@ -163,14 +163,14 @@ func (c *cmdInit) Run(cmd *cobra.Command, args []string) error {
 
 			// Add the disk.
 			req := &types.DisksPost{
-				Path:    diskPath,
+				Path:    []string{diskPath},
 				Wipe:    diskWipe,
 				Encrypt: diskEncrypt,
 			}
 
-			err = client.AddDisk(context.Background(), lc, req)
+			failures, err := client.AddDisk(context.Background(), lc, req)
 			if err != nil {
-				return err
+				return fmt.Errorf("%w\nFailed to add the following disks:\n%v", err, failures)
```

Comment:
Per convention the err variable should be spliced onto the end of the error message (and should start with lower case)

Suggest:

```
return fmt.Errorf("failed to add the following disks:\n%v \nerr: %w", failures, err)
```

Reply:
Ack Done